src/pages/Detailpage/Detailpage.js

```diff
@@ -1,13 +1,395 @@
+                    <div className="detailPageReviewText">{data.text}</div>
+                    <div className="detailPageReviewUserProfile">
+                      <div className="detailPageReviewHairProfile">
+                        <SpanTitle>hair profile: </SpanTitle>
+                        <span>{data.hairProfile}</span>
+                      </div>
+                      <div className="detailPageReviewHairGoal">
+                        <SpanTitle>goals: </SpanTitle>
+                        <span>{data.goal}</span>
+                      </div>
+                      <div className="detailPageReviewFragrance">
+                        <SpanTitle>fragrance: </SpanTitle>
+                        <span>{data.fragrance}</span>
+                      </div>
+                    </div>
+                  </div>
+                </DetailPageReview>
+              );
+            })}
+          </Slider>
+        </div>
+      </DetailPageReviewSection>
+    </div>
+  );
+};
 
 export default Detailpage;
+
+const StyledSection = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  .slick-container-test {
+    width: 600px;
+    height: 600px;
+    margin-right: 30px;
+
+    .img-container {
+      width: 600px;
+      height: 600px;
+
+      .slickImage {
+        width: 600px;
+        height: 600px;
+      }
+    }
+  }
+
+  .detailMainExplanation {
+    margin-left: 70px;
+    width: 500px;
+
+    span {
+      font-size: 16px;
+      color: #6f6f6f;
+    }
+
+    ul {
+      margin: 20px 0px 45px 0px;
+      list-style-type: disc;
+
+      li {
+        margin-bottom: 10px;
+        width: 480px;
+      }
+    }
+  }
+`;
+
+const Badges = styled.section`
+  margin: 120px 0px;
+  padding: 100px 20px 60px 20px;
+  background-color: #fffaf9;
+
+  h1 {
+    display: block;
+    align-items: center;
+    text-align: center;
+    height: 50px;
+    font-size: 32px;
+  }
+
+  div {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 40px 74px 0px 74px;
+    padding-bottom: 60px;
+
+    img {
+      margin: 8px;
+      width: 100px;
+    }
+  }
+`;
+
+const MoreInfo = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 200px;
+    width: 50%;
+    height: 700px;
+
+    div {
+      margin-left: -60px;
+      padding: 40px;
+      width: 464px;
+      height: 263px;
+
+      sup {
+        margin-left: -10px;
+      }
+      h1 {
+        padding-left: 30px;
+        font-size: 40px;
+      }
+      p {
+        margin: 16px 0px;
+        padding-left: 30px;
+        color: #878e8d;
+        font-size: 15px;
+        line-height: 30px;
+      }
+    }
+  }
+
+  img {
+    width: 50%;
+    height: 700px;
+  }
+`;
+
+const DetailPageReviewSection = styled.section`
+  margin: 50px 0px 80px 0px;
```

Comment:
아래와 같이 작성해주실 수 있습니다. 
```suggestion
  margin: 50px 0px 80px;
```

Reply:
넵 수정했습니다!